apps/extension/src/components/setting-sheet.tsx

```diff
@@ -0,0 +1,77 @@
+import {
+  HamburgerMenuIcon,
+  MixerHorizontalIcon,
+  PersonIcon,
+  SizeIcon,
+} from '@radix-ui/react-icons';
+import { Button, Sheet, SheetContent, SheetFooter, SheetHeader, SheetTrigger } from 'ui/components';
+import { useStore } from '../state';
+import { passwordSelector } from '../state/password';
+import { usePopupNav } from '../utils/navigate';
+import { PopupPath } from '../routes/popup/paths';
+import { PagePath } from '../routes/page/paths';
+
+const links = [
+  {
+    title: 'Settings',
+    icon: <MixerHorizontalIcon className='h-5 w-5 text-foreground' />,
+    href: PopupPath.SETTINGS,
+  },
+  {
+    title: 'Contact Information',
+    icon: <PersonIcon className='h-5 w-5 text-foreground' />,
+    href: PopupPath.SETTINGS_CONTACTS,
+  },
```

Comment:
`Contact information` is not something we'd have in the wallet. Think we can get rid of this.

Reply:
Makes sense to me

Reply:
but you asked in the fig if we need contact information

Reply:
there - https://www.figma.com/file/iTnIhPxUm4ZleTRKWEn4GM/Penumbra?node-id=2426%3A7424&mode=dev

Reply:
I'm confused, do we need it or not? We don't currently have contacts. Don't think it's a priority.

Reply:
Now we don't need it! Previously, we had the ability to send a transaction from the extension and we didn't remove it because we thought that maybe the opinion would change